docs/05.policy/04.groups/04.groups.md

```diff
@@ -37,6 +37,23 @@ To enable host protection with process profile rules, select the 'nodes' group a
 Network connection violations of rules shown in the Network Rules for Nodes are never blocked, even in Protect mode. Only process violations are blocked in Protect mode on nodes.
 :::
 
+#### DDoS detection
+
+User can configure following DDoS detection thresholds:
+
++ **Active Session Count**
+> Active sessions count on the group, exceeding this count can trigger a warning event
++ **Group Session Rate**
+> Connection Per Second(CPS) on the group, exceeding this rate can trigger a warning event
++ **Group Bandwidth**
+> Throughput meassured in (Mb/s) on the group, exceeding this bandwidth threshold can trigger a warning event
+
+![ddosDetection](group_ddos_threshold.png)
+
+:::note
+We sample the past 60 seconds traffic in average to see whether the metric break the threshold or not for the DDoS detection feature, measurement in protect mode is most meaningful because enforcer becomes inline at the data-path. Learned, User-created, CRD created and federal created container group can configure DDos detection thresholds.
```

Comment:
```suggestion
NeuVector samples the traffic for 60 seconds in average and checks whether the metric breaks the threshold or not.

 The DDoS detection measurement is more meaningful in protect mode because the NeuVector enforcer becomes inline at the data-path. 
 
The DDoS detection thresholds can be configured for the container groups of type `Learned`, `User-created`, `CRD created` and `federal created`.
```